### PR TITLE
Port to python3

### PR DESCRIPTION
**Pull request by kleag**
This port to python3 trains correcly with python3.6 and Tensorflow 1.12 but it fails to reproduce the paper's results. SRL F1 in test is only around 61% instead of >83%.

484000) Train loss: 0.7927    Train acc: 99.06%    Train rate:  694.6 sents/sec    Learning rate: 0.000057    Sample prob: 1.000000
        Valid loss: 2.4755    Valid acc: 90.69%    Valid rate: 2391.1 sents/sec
        log loss: 0.007826      rel loss: 0.001921      srl loss: 0.777867      trig loss: 0.005128     pos loss: 0.000000

Elapsed time: 03:06:23:54
Total time in prob_argmax: 1.316255
Total time in forward: 0.610948
Not tree: 1346
Roots < 1: 12; Roots > 1: 12; 2-cycles: -1346; n-cycles: -1346
Evaluating perl bin/eval.pl -g /home/users/gdechalendar/Projets/Decoder/srl/preprocess-conll05/conll05st-release/conll2005-dev-gold-parse.txt -s model/parse_preds.tsv
  Labeled   attachment score: 26258 / 29058 * 100 = 90.36 %
  Unlabeled attachment score: 27222 / 29058 * 100 = 93.68 %
  Label accuracy score:       27178 / 29058 * 100 = 93.53 %

Evaluating perl bin/srl-eval.pl -g /home/users/gdechalendar/Projets/Decoder/srl/preprocess-conll05/conll05st-release/conll2005-dev-gold-props.txt -s model/srl_preds.tsv
Number of Sentences    :        1346
Number of Propositions :        3248
Percentage of perfect props :  36.30

              corr.  excess  missed    prec.    rec.      F1
------------------------------------------------------------
   Overall     4093    1004    4253    80.30   49.04   60.89
----------
        A0     1175     136     906    89.63   56.46   69.28
        A1     1504     345    1490    81.34   50.23   62.11
        A2      268     120     405    69.07   39.82   50.52
        A3       29      12      85    70.73   25.44   37.42
        A4       13      10      52    56.52   20.00   29.55
        A5        0       1       2     0.00    0.00    0.00
        AA        0       0       1     0.00    0.00    0.00
    AM-ADV      113      72     166    61.08   40.50   48.71
    AM-CAU       22       5      23    81.48   48.89   61.11
    AM-DIR        5      11      31    31.25   13.89   19.23
    AM-DIS       99      33     103    75.00   49.01   59.28
    AM-EXT        1       4      27    20.00    3.57    6.06
    AM-LOC       55      49     139    52.88   28.35   36.91
    AM-MNR       71      52     171    57.72   29.34   38.90
    AM-MOD      195       5     122    97.50   61.51   75.44
    AM-NEG       53       5      51    91.38   50.96   65.43
    AM-PNC       38      34      43    52.78   46.91   49.67
    AM-PRD        1       0       2   100.00   33.33   50.00
    AM-TMP      293      81     308    78.34   48.75   60.10
      R-A0       94       9      52    91.26   64.38   75.50
      R-A1       46       5      37    90.20   55.42   68.66
      R-A2        2       0       3   100.00   40.00   57.14
  R-AM-CAU        0       1       3     0.00    0.00    0.00
  R-AM-EXT        0       0       1     0.00    0.00    0.00
  R-AM-LOC        1       4       8    20.00   11.11   14.29
  R-AM-MNR        1       3       5    25.00   16.67   20.00
  R-AM-TMP       14       7      17    66.67   45.16   53.85
------------------------------------------------------------
         V     1911      21    1337    98.91   58.84   73.78
------------------------------------------------------------

Attention UAS: 

UAS: 93.68    LAS: 90.36
POS: 96.39
SRL acc: 90.93
SRL F1: 60.89

### CODE REVIEW

**Comment by kleag** [comment]
These new commits add the possibility to analyze arbitrary text files.
